pkg/manifests/manifests_test.go

```diff
@@ -2802,6 +2811,71 @@ func TestNodeExporter(t *testing.T) {
 	}
 }
 
+func TestNodeExporterCollectorSettings(t *testing.T) {
+
+	tests := []struct {
+		name        string
+		config      string
+		argsPresent []string
+		argsAbsent  []string
+	}{
+		{
+			name:        "default config",
+			config:      "",
+			argsPresent: []string{"--no-collector.cpufreq"},
+			argsAbsent:  []string{"--collector.cpufreq"},
+		},
+		{
+			name: "enable cpufreq collector",
+			config: `
+nodeExporter:
+  collectors:
+    cpufreq:
+      enabled: true
+`,
+			argsPresent: []string{"--collector.cpufreq"},
+			argsAbsent:  []string{"--no-collector.cpufreq"},
+		},
+	}
+
+	for _, test := range tests {
+		c, err := NewConfigFromString(test.config)
```

Comment:
nit: just so it's easier to identify the test if it fails 
```suggestion
                t.Run(tc.name, func(st *testing.T) {
		    c, err := NewConfigFromString(test.config)
		    ....
		}
```

Reply:
Thanks :) just pushed a new commit using t.Run